packages/quantic/force-app/main/default/lwc/quanticSummary/quanticSummary.js

```diff
@@ -46,8 +46,8 @@ export default class QuanticSummary extends LightningElement {
     return this.state.hasQuery;
   }
 
-  get forQuery() {
-    return this.state.hasQuery ? ` for ${this.state.query}` : '';
+  get query() {
+    return this.state.hasQuery ? `${this.state.query}` : '';
```

Comment:
Does atomic have the same behavior?

Reply:
This change is consistent with the [mock ups](https://www.figma.com/file/02bJKPyf2kwP6He76B42kb/Search-page-components?node-id=69%3A99) we have.

Reply:
Is that expected that, on the first load (when the query is empty), duration is now hidden?

Reply:
oups no it got scooped up in my template tag accidentally. like a dolphin in a fishing net